src/app.css

```diff
@@ -205,6 +205,10 @@ i {
   padding: 0 15px 0 55px;
   margin: 0;
   min-height: 19px;
+  display: flex;
+}
+#log p * {
```

Comment:
We definitely do not want to use `*` selectors, pretty much ever. CSS is evaluated right-to-left, so this will select every element in the page before filtering to `p`, which is going to be really expensive. Since the direct children of paragraphs are anchors, let's filter it that way:

```css
#log p > a {
  align-items: stretch;
}
```